src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,26 @@
+package seedu.addressbook.data.person;
+
+public class Contact {
+
+    private boolean isPrivate;
+    public String value;
+
+    @Override
+    public String toString() {
+        return value;
+    }
+
+    public boolean isPrivate() {
+        return isPrivate;
+    }
+
+
+    public void setPrivacy(boolean isPrivate){
+        this.isPrivate = isPrivate;
+    }
+
+    @Override
+    public int hashCode() {
```

Comment:
In general, `equals()` and `hashCode()` methods should be overridden together. Find out why!